subsys/bluetooth/services/ias/ias_internal.h

```diff
@@ -0,0 +1,62 @@
+/** @file
+ *  @brief Internal APIs for Immediate Alert Service.
+ */
+
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#include <bluetooth/gatt.h>
+
+enum bt_ias_alert_lvl {
+	BT_IAS_ALERT_LVL_NO_ALERT,
+	BT_IAS_ALERT_LVL_MILD_ALERT,
+	BT_IAS_ALERT_LVL_HIGH_ALERT,
+};
+
+struct bt_ias_client {
+	struct bt_conn *conn;
+	struct bt_uuid_16 uuid;
+	struct bt_gatt_write_params write;
+	struct bt_gatt_discover_params discover;
```

Comment:
I think it was @MariuszSkamra who was clever and put these in a `union` as we'll probably (or should never) perform a write operation while we are doing the discovery.

Reply:
Hmm, are the `write` parameters even needed? Wasn't it write-without-response?

Reply:
Removed write parameters and added write_handle, as this is really the only thing we need for writing without a response, exactly as you said, we don't need whole write parameters structure :)